### PR TITLE
Sni limitation doc

### PR DESCRIPTION
**Pull request by zongzw**

### CODE REVIEW

**Review by zhang-shengping — APPROVED** [review]

File: docs/sni-limitation.rst [hunk 1]
```diff
@@ -1,7 +1,35 @@
...
+It can be summarized as:
+
+* Relation between certificate and SSL profile is one-to-one.
+* Certificate can be used by multiple LBs just as SSL profile can be used by multiple VSs on BIGIP side.
+
+
+SNI TLS Load Balancer
+---------------------
+
+The LB using multiple certificates called **SNI TLS Load Balancer**. Correspondingly, the triggerred VS has multiple SSL profiles.
```
Comment (zhaoqin-github) [inline docs/sni-limitation.rst]: triggerred --> triggered

File: docs/sni-limitation.rst [hunk 2]
```diff
@@ -1,7 +1,35 @@
...
+
+Among SSL profiles of a specific LB, only one of them is set with attribute "*Default SSL Profile for SNI: Enabled*".
+
+So, **Using two or more SSL profiles of "Default SSL Profile for SNI: Enabled" will cause VS creation failure.** That is the limitation of using multiple SSL profiles.
+
+But unfortunately, we have no way on OpenStack dashboard to justify wether or not a certificate has been created as SSL profile of "*Default SSL Profile for SNI: Enabled*" on BIGIP side unless we
```
Comment (zhaoqin-github) [inline docs/sni-limitation.rst]: wether --> whether

Comment (zhaoqin-github) [reply docs/sni-limitation.rst]: Suggest to change to: "But unfortunately,  OpenStack dashboard can not determine wether or not a certificate has been created as SSL profile of "*Default SSL Profile for SNI: Enabled*" on BIGIP side, unless:"

File: docs/sni-limitation.rst [hunk 3]
```diff
@@ -1,7 +1,35 @@
...
+
+* On BIGIP side, check manually that SSL profile's attribute, or
```
Comment (zhaoqin-github) [inline docs/sni-limitation.rst]: check manually --> manually check

File: docs/sni-limitation.rst [hunk 4]
```diff
@@ -1,7 +1,35 @@
...
+* Name the certificate with special hint(i.e. suffix with _sni_default).
+
+Well, they are 2 ways for solving the issue. But a better recommended way is
```
Comment (zhaoqin-github) [inline docs/sni-limitation.rst]: Not sure which is better...  "But a better recommended way is" --> "However, the recommended way is"

**Review by zhaoqin-github — CHANGES_REQUESTED** [review]
Some spelling mistake, and some minor suggestion.

File: docs/sni-limitation.rst (hunk 2 above)
Comment (zongzw) [reply docs/sni-limitation.rst]: inspired by your saying, I think the following one is better: 
But unfortunately, there is no way on OpenStack dashboard to determine whether or not a certificate has been created as SSL profile of "*Default SSL Profile for SNI: Enabled*" on BIGIP side unless

Comment (zongzw) [reply docs/sni-limitation.rst]: "OpenStack dashboard " may not be an executor/actioner for doing st.

File: docs/sni-limitation.rst (hunk 1 above)
Comment (zongzw) [reply docs/sni-limitation.rst]: done.

File: docs/sni-limitation.rst (hunk 3 above)
Comment (zongzw) [reply docs/sni-limitation.rst]: done.

File: docs/sni-limitation.rst (hunk 4 above)
Comment (zongzw) [reply docs/sni-limitation.rst]: just thinking that our 'better recommended way' is not the only one of better ways.. So I use 'a'.

**Comment by zongzw** [comment]
some replies.. FYI.

File: docs/sni-limitation.rst (hunk 4 above)
Comment (zongzw) [reply docs/sni-limitation.rst]: 'however' is better.

**Review by zhaoqin-github — APPROVED** [review]

**Pull request merged**